lib/kudos-coins.js

```diff
@@ -0,0 +1,21 @@
+// Maps the number of kudosPoints scored in a UTC calendar day to the number of
+// KudosCoins that should be awarded as a result
+
+const coinMapping = [
+  { minPoints: 25, coins: 9 },
+  { minPoints: 20, coins: 8 },
+  { minPoints: 15, coins: 7 },
+  { minPoints: 10, coins: 6 },
+  { minPoints: 2, coins: 5 },
+  { minPoints: 0, coins: 1 }
+]
```

Comment:
Order is important in the calculation so we either need to sort these as required or at least have a comment to instruct the developer when adding/removing/changing.